### PR TITLE
Display podcast

### PR DESCRIPTION
**Pull request by AliShaban1**
Implement display podcast use case and integrate with other views

### CODE REVIEW

File: src/main/java/view/DisplayPodcastsView.form
Comment (jchionak) [inline src/main/java/view/DisplayPodcastsView.form:1]: Why was this deleted? I think it's necessary for the View.

**Review by DrDabbidy — CHANGES_REQUESTED** [review]
There are some issues with this. You shouldn't be deleting the `.form` files, since they are needed to build our views. I will try to fix this up.

**Review by DrDabbidy — APPROVED** [review]
This is okay for now. We just need to consolidate for now, so I'm merging.
